scan-command/src/test/java/io/ballerina/scan/internal/Rule005Test.java

```diff
@@ -0,0 +1,35 @@
+package io.ballerina.scan.internal;
+
+import io.ballerina.projects.Document;
+import io.ballerina.scan.Issue;
+import io.ballerina.scan.RuleKind;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Rule005Test extends StaticCodeAnalyzerTest {
+    private static final String UNUSED_CLASS_PRIVATE_FIELDS = "Unused class private fields";
+
+    @Test(description = "test checkpanic analyzer")
```

Comment:
Wrong description. I would get rid of the description anyway, doesn't add value.

Reply:
Will update